blitz_api/chartjs.py

```diff
@@ -0,0 +1,201 @@
+import string
+
+from django.contrib.contenttypes.models import ContentType
+from django.core.exceptions import ObjectDoesNotExist
+from django.db.models import QuerySet, Sum, Count
+from django.db.models.functions import TruncMonth, TruncDay, \
+    TruncWeek, TruncYear
+from django.http import HttpRequest
+from rest_framework.decorators import action
+from rest_framework.permissions import IsAdminUser
+from rest_framework.response import Response
+from rest_framework import status
+from django.template.defaultfilters import date
+
+
+class ChartJSMixin(object):
```

Comment:
Comment to explain 

 - we can use this mixin in view to display chartJS
 - maybe little line of example
 - Attributs we can overwrite and what they do